pyproject.toml

```
@@ -9,7 +9,8 @@ readme = "README.md"
 python = "~3.10.0"
 python-dotenv = "*"
 openai = "*"
-langchain = "^0.1.1"
+# ToDo - Update to 2.0
+langchain = "0.2.6"
```

Avoid strict dependency requirements. Also, why the TODO?
```suggestion
langchain = "^0.2.6"
```

---

good catch - also removed todo